incubator/kube-registry-proxy/values.yaml

```diff
@@ -0,0 +1,8 @@
+image:
+  repository: gcr.io/google_containers/kube-registry-proxy
+  tag: 0.4
```

Comment:
Yeah, someone from the k8s maintainer team has got to push the new release (pinging @roberthbailey from https://github.com/kubernetes/kubernetes/pull/35797)